tests/components/aurora/test_config_flow.py

```diff
@@ -0,0 +1,113 @@
+"""Test the Aurora config flow."""
+
+from homeassistant import config_entries, data_entry_flow, setup
+from homeassistant.components.aurora.const import DOMAIN
+
+from tests.async_mock import patch
+from tests.common import MockConfigEntry
+
+DATA = {
+    "name": "Home",
+    "latitude": -10,
+    "longitude": 10.2,
+}
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.aurora.config_flow.AuroraForecast.get_forecast_data",
+        return_value=True,
+    ), patch(
+        "homeassistant.components.aurora.async_setup", return_value=True
+    ) as mock_setup, patch(
+        "homeassistant.components.aurora.async_setup_entry",
+        return_value=True,
+    ) as mock_setup_entry:
+        result2 = await hass.config_entries.flow.async_configure(
+            result["flow_id"],
+            DATA,
+        )
+
+    assert result2["type"] == "create_entry"
+    assert result2["title"] == "Aurora - Home"
+    assert result2["data"] == DATA
+    await hass.async_block_till_done()
```

Comment:
The block till done should be moved to within the patch context manager.

Reply:
Fixed this comment and the bottom one here:
https://github.com/home-assistant/core/pull/43128